rootfs/mysql2storage

```diff
@@ -17,7 +17,11 @@ if [[ -v "AWS_ACCESS_KEY_ID" ]]; then
 	s3filename="${AWS_S3_FILE_PREFIX}-$(date +%s).sql.gz"
 
 	echo "[$(date -Iseconds)] Started s3 upload"
-	aws s3 cp "${filename}.gz" "s3://${AWS_S3_BUCKET}/${s3filename}"
+        if [[ -v "AWS_ENDPOINT_URL" ]];then
```

Comment:
Please use tab to indent as previous line

Reply:
done